_profiles/drupal/resource-versioning/index.md

```diff
@@ -0,0 +1,301 @@
+  - name: Gabriel Sullice
+    email: gabriel@sullice.com
+  - name: Mateu Aguiló Bosch
+    email: mateu@mateuaguilo.com
+    website: https://mateuaguilo.com
+  - name: Wim Leers
+    email: work@wimleers.com
+    website: https://wimleers.com
+
+categories:
+  - Resource Versioning
+---
+  
+# Concepts
+Resources on a server may undergo changes and the state of a resource with an
+arbitrary number of changes may be accessible. It is often useful to retrieve
+resources as they existed at the time of their creation or in various states of
+change for editorial or archival purposes. This profile establishes a protocol
+for accessing resources in those various states.
+
+For the purpose of understanding this profile, it is helpful to establish a
+vocabulary for describing various possible states of a resource.
+
+A _revision_ is to be understood as an identifiable state of a resource after
+its creation or after some number of changes.
+
+A _version_ is to be understood as a revision that is or was the default
+revision of a resource. In other words, as a revision of a resource that is
+available, or was previously available, without any version negotiation.
+
+A _working copy_ is the revision to which new changes can be made or to which
+they will be applied. Colloquially, a working copy is often thought of as the
+"tip" of a version history.
+
+For example, a resource which existed as a draft and underwent multiple changes
+before the resource was published may have many revisions. Only the published
+revision would be considered a version. The revisions prior to first version
+would have been known as the working copy as each one was created.
+
+If the version is then checked out for further changes each new revision becomes
+the working copy. When the working copy is checked in and made the default
+revision, this revision becomes the latest version and the version before it is
+its _predecessor version_. If no other revisions come after the latest version,
+it may also be considered the working copy.
+
+Sophisticated servers may support multiple versions of a resource as well as
+multiple working copies of a resource (e.g., to support multiple
+languages or within a version control system which supports multiple branches).
+
+This profile creates a standard for JSON:API to support these diverse versioning
+schemes.
+# Query Parameter
+
+## Usage
+
+An endpoint **MAY** support a `resourceVersion` query parameter to allow a
+client to indicate which version(s) of a resource should be returned.
+
+If an endpoint does not support the `resourceVersion` parameter, it **MUST**
+respond with `400 Bad Request` to any requests that include it. An error object
+detailing the source of the error **SHOULD** include a `source` member referencing
+the `resourceVersion` query parameter.
+
+If an endpoint supports the `resourceVersion` parameter and a client supplies
+it:
+
+  - The server’s response **MUST** contain the most appropriate version of the
```

Comment:
I think "the most appropriate" is unnecessarily vague. Can't we change `the most appropiate version` to `the requested version`?